tests/acceptance/course-page/code-examples/vote-test.js

```
@@ -94,7 +94,7 @@ module('Acceptance | course-page | code-examples | vote', function (hooks) {
     await coursePage.codeExamplesTab.solutionCards[0].downvoteButton.hover();
 
     assertTooltipContent(assert, {
-      contentString: "Downvote this example and we'll show it less often.",
+      contentString: "Your feedback helps us identify examples that need review.",
```

ESLint

[error] Replace `"Your·feedback·helps·us·identify·examples·that·need·review."` with `'Your·feedback·helps·us·identify·examples·that·need·review.'`

(prettier/prettier)